### DISCUSSION TITLE
Is there a way to show time & date of the last update of friends and mine? (Android)

### DISCUSSION THREAD

**Discussion by fooryo** [owntracks/android#1902] [Q&A]
I use owntracks with MQTT server.
I'm asking if there is a way to show time and date of the last update of a friend directly in the app (android)

**Comment by jpmens** [comment]
The time is shown on the right in the list of friends, but I don't see that the date is displayed.

**Reply by fooryo** [reply]
> The time is shown on the right in the list of friends, but I don't see that the date is displayed.

Yes, you see the time if the date is equal to today date. You will see the date (but not the time) if last update happened any day before today.

I'm asking if there is a way to show time + date no matter what.

**Comment by jpmens** [comment]
I don't believe there is. You might want to open an issue with an enhancement request.

**Reply by jpmens** [reply]
we should be able to put that on the Friend's details slider.

**Comment by jpmens** [comment]
https://github.com/owntracks/android/issues/1903
